### ISSUE TITLE
0.62 breaks dynamic enums

### ISSUE THREAD

**Issue by orhanhenrik** [BoundaryML/baml#1079]
After upgrading to 0.62, running baml beaks with the following error:
```
<enum 'FeeType'> has no members; specify `names=()` if you meant to create a new, empty, enum
```

I construct my `tb` object like this:
```python
    tb = type_builder.TypeBuilder()  # type: ignore[no-untyped-call]
    for value in MyPythonEnum:
        tb.FeeType.add_value(value)
```

And in my .baml file I have
```
enum FeeType {
  // Populated from enum in Python
  @@dynamic
}
```

Code runs fine on 0.61.1

**Comment by hellovai** [comment]
oh shoot!  Thank you for flagging this @orhanhenrik 

This a great catch. We somehow didn't have this in our integ tests. I will add a test and release a patch for this ASAP!

**Comment by hellovai** [comment]
This is now fixed in Release 0.63.0+
